packit_service/service/api/webhooks.py

```diff
@@ -230,7 +240,7 @@ def validate_token(self):
         token = request.headers["X-Gitlab-Token"]
 
         if token in config.gitlab_webhook_tokens:
-            logger.debug("Deprecation Warning: Old Gitlab tokens used.")
+            logger.debug("Deprecation Warning: Old/static Gitlab tokens used.")
```

Comment:
Do we want to get rid of those "static" ones?

I treat them more like a different use-case, but:
* After this PR, I am not sure if we can find some use-case.
* Looks like having two ways is confusing to people.

Reply:
I'd keep it there for some more time until we prove that this new group webhook tokens work OK.